src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -264,10 +271,52 @@ private int LineBreaksAfter(SyntaxToken currentToken, SyntaxToken nextToken)
             return 0;
         }
 
+        private static bool IsAccessorListWithoutAccessorsWithBlockBody(SyntaxNode? node)
+            => node is AccessorListSyntax accessorList &&
+                accessorList.Accessors.All(a => a.Body == null);
+
+        private static bool HasInitializer(BasePropertyDeclarationSyntax basePropertyDeclaration)
+            => basePropertyDeclaration is PropertyDeclarationSyntax property && property.Initializer != null;
+
+        private static bool IsAccessorListFollowedByInitializer(SyntaxNode? node)
+            => node is AccessorListSyntax accessorList &&
+                accessorList.Parent is BasePropertyDeclarationSyntax basePropertyDeclarationSyntax &&
+                HasInitializer(basePropertyDeclarationSyntax);
```

Comment:
```suggestion
            => node is AccessorListSyntax &&
                node.Parent is PropertyDeclarationSyntax property &&
                property.Initializer != null;
```

Reply:
@CyrusNajmabadi  Are you sure about this?
This prevents any initializers outside the properties (if they will ever exists)

Reply:
@AlekseyTs what do you think about this?
I was searching to avoid looking for properties on purpose so that any derived class from `BasePropertyDeclarationSyntax` having an accessor fit in the inline formatting.

Reply:
> what do you think about this?

I am fine with the code and names as it is.

---
In reply to: [534376783](https://github.com/dotnet/roslyn/pull/49495#discussion_r534376783) [](ancestors = 534376783)

Reply:
> This prevents any initializers outside the properties (if they will ever exists)

your existing code already prevents taht as you do a type check for PropertyDeclarationSyntax yourself in HasInitializer.

Reply:
> your existing code already prevents taht as you do a type check for PropertyDeclarationSyntax yourself in HasInitializer.

Sure, but is well identified in that point. Currently I can't do differently because properties arei the only one allowed.
Could in the future be possible, let's say, for indexers to be initialized, the changes would be limited to a single point and this function would work correctly.
I don't see how the current version could cause problems. Let me know if you want me to change it anyway, so that I can send the commit before night (here). Thanks

Reply:
> the changes would be limited to a single point and this function would work correctly.

that is true with the change i suggested as well.  in both cases we will need to update the code.  so might as well have hte code be correct and precise for the trees we have today.  

> Let me know if you want me to change it anyway

Yes please :)

Reply:
Okay, and what about the method names?

Reply:
i'll defer to Aleksey's judgement on those :)

Reply:
@AlekseyTs I committed all the required changes but not the method renaming.
It's up to you now :)